.goreleaser.yml

```diff
@@ -1,7 +1,11 @@
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
```

Comment:
Could you explain why you disable `CGO`?

Reply:
I remove it because we are not using

Reply:
thanks for the reply! I want to mention that [pebble](https://github.com/cockroachdb/pebble) has different behavior internally depending on whether CGO is enabled or not.  I know this because when we started the hermez-node we used a version of pebble that had a bug when CGO was disabled!  The bug was fixed in November 2020.  See here for mentions of CGO in pebble: https://github.com/cockroachdb/pebble/search?q=CGO&type=issues

I'm mentioning this to be aware that one of our dependencies is "sensitive" to CGO, to consider this in the future if any issue occurs!